simpler-wick.sty

```diff
@@ -255,50 +255,17 @@
   \fi
 }
 
-% L. 
-% above routine for the 'bottom' wick contraction command
-\def\bwick{
-	\@ifnextchar[{
-		\bwick@
-	}{
-		\bwick@[]
-}}
 
-\def\bwick@[#1]#2{
-	\ifmmode
-	\begingroup
-	\pgfkeys{
-		simplerwick,
-		#1}
-	% Define the variables and commands
-	\swick@cond@reset
-	\swick@count=0
-	\def\swick@max{0}
-	\def\b{\swick@sbcon} % this doesn't have to be \b, originally I had defined it so it could be used in the above environment, however
-  % the vertical spacing is not right due to the vbox, so I had to create a new environment so the distinction is no longer necessary 
-  % this might be resolved with some conditional statements, but this works for now.
+% L. Remant section, 
```

Comment:
The remaining code here is just commented out, so it could be remove from the pull request.